packages/db/tests/translate/test-granules.js

```diff
@@ -244,6 +244,7 @@ test('translatePostgresGranuleToApiGranule converts Postgres granule to API gran
     providerPgModel,
     filePgModel,
   });
+  result.files.sort((a, b) => (a.fileName > b.fileName ? 1 : -1));
```

Comment:
I grok *why* this was needed, but in the context of this PR, why was it needed?     Intermittent failing units?

Reply:
edit: answer, because file prefix was updated and caused/exposed this behavior.

Reply:
It wasn't because of the file-prefix.  the file order isn't guaranteed and the tests were only passing on CI because of how my database was filled.  So this just needed to be updated so that it will work irrespective of what order they are in in the returned granule.